crates/router/Cargo.toml

```diff
@@ -120,6 +120,8 @@ openapi = { version = "0.1.0", path = "../openapi", optional = true }
 erased-serde = "0.3.31"
 quick-xml = { version = "0.31.0", features = ["serialize"] }
 rdkafka = "0.36.0"
+isocountry = "0.3.2"
```

Comment:
This is not necessary, As we are already using iso codes in our enum

Reply:
this is required to get numeric_id of the country